x/evm/keeper/msg_server.go

```diff
@@ -280,7 +280,9 @@ func (s msgServer) Link(c context.Context, req *types.LinkRequest) (*types.LinkR
 			sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
 			sdk.NewAttribute(types.AttributeKeyChain, senderChain.Name),
 			sdk.NewAttribute(types.AttributeKeyBurnAddress, burnerAddr.Hex()),
+			sdk.NewAttribute(types.AttributeKeyDepositAddress, burnerAddr.Hex()),
 			sdk.NewAttribute(types.AttributeKeyAddress, req.RecipientAddr),
+			sdk.NewAttribute(types.AttributeKeyDestinationAddress, req.RecipientAddr),
 			sdk.NewAttribute(types.AttributeKeyDestinationChain, recipientChain.Name),
```

Comment:
in the same manner, these should be recipient chain and address instead of destination

Reply:
it's already at destinationChain and destinationAddress in all places (including in the axelnet module). Ok if we just keep it as destination[Val] accordingly?